client/src/components/AdminContainer/index.js

```diff
@@ -48,6 +49,16 @@ const AdminContainer = ({ children, buttonContent, buttonRoute, logout }) => {
                 </li>
               </ul>
               <div className="admin-side-btn">
+                {history.location.pathname !== '/admin' ? (
+                  <Button
+                    className="admin-back-btn"
+                    type="primary"
+                    onClick={() => history.goBack()}
```

Comment:
@rehabas You might be able to use goBack directly, without an arrow function: `onClick={history.goBack}`